servicetalk-data-protobuf/src/main/java/io/servicetalk/data/protobuf/ProtobufSerializerFactory.java

```diff
@@ -54,6 +60,18 @@ private ProtobufSerializerFactory() {
         return serializerMap.computeIfAbsent(parser, parser2 -> new ProtobufSerializer<>((Parser<T>) parser2));
     }
 
+    /**
+     * Get a {@link SerializerDeserializer}.
+     * @param clazz Used to obtain a {@link Parser} which is used to serialize and deserialize.
+     * @param <T> The type to serialize and deserialize.
+     * @return a {@link SerializerDeserializer}.
+     */
+    public <T extends MessageLite> SerializerDeserializer<T> serializerDeserializer(Class<T> clazz) {
+        @SuppressWarnings("unchecked")
+        final Parser<T> parser = (Parser<T>) parserMap.get(clazz);
```

Comment:
There is a race here than can result in more than on instance of `Parser<T>` for a given class, but normally just the one cached instance.

Reply:
The race is benign but I can use `computeIfAbsent`. I was previously throwing the checked exceptions which made this more painful. The `computeIfAbsent` will just require more casting (on generator and return value) but should be fine.